### ISSUE TITLE
Description of what each settings does (Tooltip)

### ISSUE THREAD

**Issue by ghost** [LucasGGamerM/moshidon#169]
The ability to long press on each setting to show tooltip on what it actually does which reduce the need of long sentence and put in tooltip for clear description.

Eg: "Show interaction counts" >long press will show a tooltip "Show the reblog, comment, favorite counts in timeline and anywhere"

Eg: "Publish button text" >long press will show a tooltip "Change the Publish button to any text you want; eg: Toot"

Eg: "Uniform icons for all notification" > long press will show a tooltip "Show the matching Moshidon icons for all notification. Disable it to see shiny icons. Eg: Star icons for favorite, bubble for replies etc..."

Those are just few examples.

**Comment by FineFindus** [comment]
While the overall idea is good, and it was already discussed for the settings design, to feature subtitle descriptions, it's a terrible idea showing them in the tooltips.
[Tooltips](https://m2.material.io/components/tooltips#usage) should only be used to show basic information, such as descriptions for an icon, e.g. a printer icon would show the print label. Showing full on description would be totally overall. Additionally, long tooltips are hard to read, as the user has to continuously press the touch surface and only one tooltip would be visible at the time, as well as being hard to discover.

In summary, the would be a misuse of the design element, for which it is subotimal when better alternatives exist, which should be used.

**Comment by FineFindus** [comment]
As I said, I agree with the general idea, I just disagree with how it should be done.
A better alternative is to add subtitles to the setting rows, which is already supported by material design.
